elixir/config/config.exs

```diff
@@ -305,9 +301,15 @@ config :workos, WorkOS.Client,
   baseurl: "https://api.workos.com"
 
 config :sentry,
-  dsn: "https://29f4ab7c6c473c17bc01f8aeffb0ac16@o4507971108339712.ingest.us.sentry.io/4508756715569152",
+  # Setting this to nil will disable reporting
+  dsn:
+    "https://29f4ab7c6c473c17bc01f8aeffb0ac16@o4507971108339712.ingest.us.sentry.io/4508756715569152",
```

Comment:
Do we need to set this in an ENV variable?  Will this cause us to ingest anything coming from self-hosting customers and/or local dev instances?

Reply:
Hm I had overridden in `runtime.exs`, but yeah I think it's better the other way around.

The elixir SDK actually has great test integration so we could theoretically enable it with `test_mode` and write tests that certain events make it to Sentry.